frontend/src/features/manageVehicles/hooks/endpoints/endpoints.tsx

```diff
@@ -0,0 +1,12 @@
+import { getOpenshiftEnv } from "../../../../common/helpers/helpers";
+
+const ENV = getOpenshiftEnv();
+
+export const MV_BASE_URL =
+  ENV === "localhost"
+    ? "http://localhost:5000"
+    : `https://onroutebc-${ENV}-backend.apps.silver.devops.gov.bc.ca`;
```

Comment:
if `getOpenshiftEnv` returns `other`, this url will resolve to `onroutebc-other-backend`. Is this correct?

Reply:
Agreed. I have made a temporary fix. 

See this discussion for a future solution: [#95 ](https://github.com/bcgov/onroutebc/pull/95#discussion_r1059515065)